### PR TITLE
Update translation_RU.json. Fixed a phrase that was too long causing …

### PR DESCRIPTION
**Pull request by NunkyJoe**
…temperature values to not fit on the sleep screen.

The sleep indication phrase in simple mode is too long "Хххррп". The temperature value does not fit on the screen. Replaced it with the original phrase "Zzzz", understandable in all languages.

* **Please check if the PR fulfills these requirements**
- [] The changes have been tested locally
- [] There are no breaking changes

* **What kind of change does this PR introduce?**

* **What is the current behavior?**

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Comment by ia** [comment]
First, thanks to point out the issue: my bad was not to check all the strings for all modes & screens, sorry.

Second,
> Replaced it with the original phrase "Zzzz", **understandable in all languages**.

the last part goes a bit more tricky the more I think about.

The point of translation (in general) - not only translation but adaptation. So I just thought it will be witty to put a string representing a sound of snoring as heard by Eastern Europeans usually, since that's what "z" for _EN_. Because for some other translations in _IronOS_ (like for _ES_ or _SR_CYR_) there are adaptations (of some sort) for this string already.

TL;DR:
- if we do not only translation but _"localization"_ as well, then the string should be just shorten like: Ххрп;
- if we do _"globalization"_ & _unification_, then the string should be indeed _Zzzz_, but at the same time this string _Zzzz_ should be the same for every other language in _Translations/_ as well (the latter can be done later, when we figure out what to do here exactly).

Personally, I still would like to stick to the first option. But I really want to know what others have to say on that.

P.S. Happy New Year! :tada:

**Review by discip — APPROVED** [review]
